includes/class-wc-subscriptions-cart.php

```diff
@@ -105,6 +105,7 @@ public static function init() {
 		// When WooCommerce determines the taxable address only return pick up shipping methods chosen for the recurring cart being calculated.
 		add_filter( 'woocommerce_local_pickup_methods', __CLASS__ . '::filter_recurring_cart_chosen_shipping_method', 100, 1 );
 		add_filter( 'wc_shipping_local_pickup_plus_chosen_shipping_methods', __CLASS__ . '::filter_recurring_cart_chosen_shipping_method', 10, 1 );
+		add_action( 'woocommerce_store_api_cart_select_shipping_rate', __CLASS__ . '::set_recurring_cart_chosen_shipping_method', 10, 2 );
```

Comment:
This hook `woocommerce_store_api_cart_select_shipping_rate` isn't executing for me. I'm not sure what I'm missing.

I attempted searching for it with no luck on WC/WC Blocks. Is it a dynamic hook?

My env:
WP 6.1
WC 7.1.0 (also tried trunk)
WC Blocks branch [add/local-pickup-tax-locations](https://github.com/woocommerce/woocommerce-blocks/tree/add/local-pickup-tax-locations) (tried `npm run build` and `npm run dev`)
WC Subs 4.6.0
`define('JETPACK_AUTOLOAD_DEV', true);` in `wp-config.php`

Reply:
It should be here now if you want to checkout again: https://github.com/woocommerce/woocommerce-blocks/pull/7433/files#diff-36db0c99c9ec11b21a09848d8335af16df6aa687f32414484df6363b4e707051R97